keyboards/keychron/k3_pro/ansi/white/keymaps/vial/vial.json

```
@@ -0,0 +1,253 @@
+{
+    "name": "Keychron K3 Pro ANSI White",
+    "vendorId": "0x3434",
+    "productId": "0x0233",
+    "lighting": "qmk_backlight",
```

```suggestion
    "lighting": "none",
```
This variant uses LED_Matrix, and there's no lighting option for it for `vial.json`.